build_docs.sh

```diff
@@ -46,11 +46,10 @@ if which gimli >/dev/null; then
 	for i in "${doc_files[@]}"
 	do
 		cat "$i" >> ${filename}.md
+
 	done
-	rm -f ${filename}.pdf
-	gimli -f ${filename}.md -stylesheet override.css \
-	  -w '--toc --title "Cleanflight Manual" --footer-right "[page]" --toc-depth 1'
-	rm ${filename}.md
+        gimli -f {filename}.md -stylesheet override.css \
```

Comment:
This needs to be `${filename}.md`, or the file (`Manual.md`) won't be picked up.

Reply:
Yeah, I can't figure out why it works without the '$' sign. It doesn't make sense, maybe it's due to how gimli works behind the scenes? Would you like a youtube/snapchat clip as proof that it works? An email of the pdf?

Reply:
Well, it needs to work for _everybody_. 'Works on my machine' is not enough to make this useful.

Omitting the `$` is clearly wrong, this will just get gimly to try and use the (non-existing) file `{filename}.md` as input.

Reply:
There, I think I fixed it.  Thank you for your patience.